morecantile/commons.py

```diff
@@ -2,10 +2,9 @@
 
 from collections import OrderedDict, namedtuple
 
-from rasterio.coords import BoundingBox  # noqa
-
 _Tile = namedtuple("_Tile", ["x", "y", "z"])
 _Coords = namedtuple("_Coords", ["x", "y"])
+BoundingBox = namedtuple("BoundingBox", ("left", "bottom", "right", "top"))
```

Comment:
maybe it's also time to move away from nametuple 

cc @kylebarron

Reply:
If you stick with `namedtuple`, I like the new class-like syntax of

```py
from typing import NamedTuple

class BoundingBox(NamedTuple):
	left: int
	bottom: int
	right: int
	top: int
```

But that's a personal preference 🤷‍♂️